scripts/background.js

```diff
@@ -177,7 +159,7 @@ chrome.webRequest.onCompleted.addListener(function (details) {
     var httpFailCodes = [404, 408, 410, 451, 500, 502, 503, 504, 509, 520, 521,
```

Comment:
Everything with status >= 400 is a failure. Why do we check for these status codes explicitly?